test/server/cards/03_TWI/events/BloodSport.spec.ts

```diff
@@ -0,0 +1,64 @@
+describe('Blood Sport', function() {
+    integration(function(contextRef) {
+        describe('Blood Sport\'s ability', function() {
+            describe('when ground arena is not empty', function() {
+                beforeEach(function () {
+                    contextRef.setupTest({
+                        phase: 'action',
+                        player1: {
+                            hand: ['blood-sport'],
+                            groundArena: ['atst'],
+                            spaceArena: ['cartel-spacer'],
+                            leader: { card: 'boba-fett#daimyo', deployed: true }
+                        },
+                        player2: {
+                            groundArena: ['wampa'],
+                            spaceArena: ['alliance-xwing'],
+                            leader: { card: 'luke-skywalker#faithful-friend', deployed: true }
+                        }
+                    });
+                });
+
+                describe('when played with targets', function() {
+                    it('should deal 2 damage to each ground unit', function () {
+                        const { context } = contextRef;
+
+                        context.player1.clickCard(context.bloodSport);
+
+                        expect(context.atst.damage).toBe(2);
+                        expect(context.cartelSpacer.damage).toBe(0);
+                        expect(context.wampa.damage).toBe(2);
+                        expect(context.allianceXwing.damage).toBe(0);
+                        expect(context.bobaFett.damage).toBe(2);
+                        expect(context.lukeSkywalker.damage).toBe(2);
+                    });
+                });
+            });
+
+            describe('when ground arena is empty', function() {
+                beforeEach(function () {
+                    contextRef.setupTest({
+                        phase: 'action',
+                        player1: {
+                            hand: ['blood-sport'],
+                            groundArena: [],
+                            spaceArena: ['cartel-spacer']
+                        },
+                        player2: {
+                            groundArena: [],
+                            spaceArena: ['alliance-xwing']
+                        }
+                    });
+                });
+
+                it('does nothing', function () {
```

Comment:
This is a good test case but since we're trying to keep the numbers down due to long run time we can omit this test, we have a lot of cases checking that an event still plays if it has no targets